tests/providers/tpm.rs

```diff
@@ -284,3 +321,61 @@ fn asym_encrypt_with_crate() {
 
     assert_eq!(&initial_plaintext[..], &plaintext[..]);
 }
+
+#[test]
+fn import_private_key() {
```

Comment:
Thanks for adding the test!
@ionut-arm is it fine for this test to be here or should we add it as an e2e test?

Reply:
Leave it there! I'll rebase my changes, though I won't finish today